src/js/components/calendar.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import CalendarJS from 'calendarjs';
+import Month from './month.jsx';
+
+export default class Calendar extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentDate: moment(props.initialDate),
+      selectedTimeslot: null,
+    };
+  }
+
+  render() {
+    return (
+      <div className = "tsc-calendar">
+        { this._renderActions() }
+        { this._renderMonth() }
+      </div>
+    );
+  }
+
+  _renderActions() {
+    const {
+      currentDate,
+    } = this.state;
+
+    const actionTitle = `${currentDate.format('MMMM - YYYY')}`;
+
+    return (
+      <div className = "tsc-calendar__actions">
+        <div className = "tsc-calendar__action tsc-calendar__action-element tsc-calendar__action-element--left" onClick = { this._onGoToPrevMonth.bind(this) }>
+          &#8249;
+        </div>
+        <div className = "tsc-calendar__action tsc-calendar__action-title">
+          { actionTitle }
+        </div>
+        <div className = "tsc-calendar__action tsc-calendar__action-element tsc-calendar__action-element--right" onClick = { this._onGoToNextMonth.bind(this) }>
+          &#8250;
+        </div>
+      </div>
+    );
+  }
+
+  _renderMonth() {
+    const {
+      currentDate,
+    } = this.state;
+
+    const cal = new CalendarJS(currentDate.year(), currentDate.month() + 1);
```

Comment:
Either use **Calendar** or **CalendarJS**. Other files (tests, for example) use **Calendar** when importing calendarjs. So choose the most suitable one (I personally prefer CalendarJS).